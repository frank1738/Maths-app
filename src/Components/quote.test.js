import React from 'react';
import Quote from './Quote';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
