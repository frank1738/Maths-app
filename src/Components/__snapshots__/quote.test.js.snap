// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="home-section quote"
  id="quote"
>
  " Mathematics is not about numbers,equations,computations,or algorithms: it is about understanding." -William Paul Thurston
</section>
`;
